client/pkg/omnictl/talosconfig.go

```diff
@@ -140,9 +142,10 @@ func init() {
 	talosconfigCmd.Flags().StringVarP(&talosconfigCmdFlags.cluster, "cluster", "c", "", "cluster to use")
 	talosconfigCmd.Flags().BoolVarP(&talosconfigCmdFlags.force, "force", "f", false, "force overwrite of talosconfig if already present")
 	talosconfigCmd.Flags().BoolVarP(&talosconfigCmdFlags.merge, "merge", "m", true, "merge with existing talosconfig")
+	talosconfigCmd.Flags().BoolVar(&talosconfigCmdFlags.getAdminConfig, "admin", false, "get admin talosconfig (if enabled)")
```

Comment:
Naming question: should we call it simply "admin kubeconfig"? It kinda implies that it is part of a regular workflow and not something reserved for break-glass scenarios.